keps/sig-apps/3335-statefulset-slice/README.md

```diff
@@ -448,12 +451,18 @@ For Beta and GA, add links to added tests together with links to k8s-triage for
 https://storage.googleapis.com/k8s-triage/index.html
 -->
 
+`Feature:StatefulSetStartOrdinal` in `k8s.io/kubernetes/test/e2e/apps/`.
+
+*   Adding `ordinals.start`: Validate that setting `ordinals.start` to `k` causes StatefulSet ordinals to be scaled (pods `[0, k-1]` are terminated, pods `[N, N+k-1]` are created)
```

Comment:
Same here - no need to describe what exactly the test is using. Do what the comment is saying:

For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
https://storage.googleapis.com/k8s-triage/index.html

Reply:
Same for integration

Reply:
Added links to the Testgrid and the k8s-triage (however it appears k8s-triage only show failures, so the test regex can be used to quickly identify if a test with this name is failing).